Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
"to condescend" can be translated to: "a condescender" o "para condescender", depending of the context of the original phrase.

"comprueba que la RDB esté actualizada" is ok, just that sounds different than english in context of the phrase (personal opinion), anyway we can modify the english phrase to: "check that the RDB is up-to-date" and it will match perfectly I guess.